tests/Paulus/Tests/Exception/HttpExceptionFactoryTest.php

```diff
@@ -0,0 +1,109 @@
+<?php
+/**
+ * Paulus - A PHP micro-framework for creating RESTful services
+ *
+ * @author      Trevor Suarez (Rican7)
+ * @copyright   2013 Trevor Suarez
+ * @link        https://github.com/Rican7/Paulus
+ * @license     https://github.com/Rican7/Paulus/blob/master/LICENSE
+ * @version     2.0.0
+ */
+
+namespace Paulus\Tests\Exception;
```

Comment:
The namespace of this test should mirror the namespace nesting of the class its testing, so this should be: `Paulus\Tests\Exception\Http\Standard`. :)